landoapi/transplants.py

```diff
@@ -51,16 +65,115 @@
 CODE_FREEZE_OFFSET = "-0800"
 
 
+@dataclass
+class LandingAssessmentState:
+    """Encapsulates the state of a landing request for assessment.
```

Comment:
Docstring should describe what a "state of a landing request" means. Also, perhaps what a "landing request" is and how it is different than a transplant.

I think "landing request" might be a bit confusing because technically a landing job is a landing request (i.e., a request to land something). Is there a better name for this?